vllm/model_executor/models/interfaces_base.py

```diff
@@ -62,7 +59,7 @@ def _check_vllm_model_forward(model: Union[Type[object], object]) -> bool:
     if not callable(model_forward):
         return False
 
-    vllm_kws = ("input_ids", "positions", "kv_caches", "attn_metadata")
```

Comment:
same suggestion here, it will be better to use seperate PR or commit to do the API change